scl_prefixes.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
Would you entertain yaml as a format instead? I find it less cumbersome for human edits than json.

Reply:
I typically choose JSON because it's easier to handle in pure python where YAML is an extension. I originally intended to treat this as a cache where I'd fill it from repositories.

Reply:
Does the YAML extension require installing extra packages with pip? Is that the reason?

Reply:
Yes, it's a pip package. That's why often go for JSON. Especially since in this case it's not really a complex structure that often changes.